### PR TITLE
Split 02-numpy.md

### PR DESCRIPTION
**Pull request by maxim-belkin**
Extracting visualization content into its own episode.

Once approved, this PR has to be merged with a merge commit.

### CODE REVIEW

**Comment by ldko** [comment]
README.md should also be updated to reflect the new episode and re-numbered episodes.

File: _episodes/03-matplotlib.md [hunk 1]
```diff
@@ -0,0 +1,256 @@
+---
+title: Analyzing Patient Data
```
Comment (ldko) [inline _episodes/03-matplotlib.md]: This title should be changed.

**Comment by maxim-belkin** [comment]
Thanks, Lauren!

File: _episodes/03-matplotlib.md (hunk 1 above)
Comment (maxim-belkin) [reply _episodes/03-matplotlib.md]: Updated!

File: _episodes/03-matplotlib.md [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+teaching: 30
+exercises: 20
+questions:
+- "How can I visualize tabular data in Python?"
+- "How can I group several plots together?"
+objectives:
+- "Plot simple graphs from data."
+- "Group several graphs in a single figure."
+keypoints:
+- "Use the `pyplot` library from `matplotlib` for creating simple visualizations."
```
Comment (annefou) [inline _episodes/03-matplotlib.md]: below we say "pyplot module from matplotlib library". Maybe we could use the same here. What do you think?

Comment (maxim-belkin) [reply _episodes/03-matplotlib.md]: Thanks, Anne. I think it makes sense to use the same wording.

**Comment by maxim-belkin** [comment]
Anne, Lauren: Thank you for reviewing this. Please post any additional comments if you have them. I'd like to merge this by the COB today if you think it is ready.

**Pull request merged**